CHANGELOG

```diff
@@ -1,3 +1,16 @@
+Version 0.2.0 (2017-04-XX)
+--------------------------
+Use standard regular expression for schema URIs (#22)
+Migrate usage guide from README to dedicated snowplow/snowplow wiki page (#30)
+Update README markdown in according with CommonMark (#28)
+Migrate setup guide from README to dedicated snowplow/snowplow wiki page (#29)
+Add support for checking and setting a DynamoDB-backed run manifest (#31)
+Allow empty custom contexts (#27)
+Replace Scalaz Validation with Scala Either (#20)
+Bump specs2 to 3.8.9 (#33)
+Add CI/CD to project (#18)
+Bump SBT to 0.13.15 (#32)
```

Comment:
Can we change the ordering here? I'd start with bumps, then new features, then CI/CD and documentation